src/badges/tests/test_views.py

```diff
@@ -68,15 +68,15 @@ def test_all_badge_page_status_codes_for_students(self):
         # students shouldn't have access to these and should be redirected
 
         self.assertRedirectsQuests('badges:badge_create', follow=True),
```

Comment:
How come this one redirects to quests and the others are all 403?  Seems like this should be 403 too.

Reply:
The view linked to the url was manually redirecting to ``quest_manager:quests``.

Fixed it + removed huge comment about why they made it like that